virtual-desktop/plugin-config/tslint.base.linient.json

```diff
@@ -0,0 +1,168 @@
+{
+  "rules": {
+    "adjacent-overload-signatures": true,
+    "align": {
+      "options": ["parameters", "statements"]
+    },
+    "array-type": {
+      "options": ["array-simple"]
+    },
+    "arrow-parens": true,
+    "arrow-return-shorthand": true,
+    "ban-types": {
+      "options": [
+        ["Object", "Avoid using the `Object` type. Did you mean `object`?"],
+        [
+          "Function",
+          "Avoid using the `Function` type. Prefer a specific function type, like `() => void`."
+        ],
+        ["Boolean", "Avoid using the `Boolean` type. Did you mean `boolean`?"],
+        ["Number", "Avoid using the `Number` type. Did you mean `number`?"],
+        ["String", "Avoid using the `String` type. Did you mean `string`?"],
+        ["Symbol", "Avoid using the `Symbol` type. Did you mean `symbol`?"]
+      ]
+    },
+    "callable-types": true,
+    "class-name": true,
+    "comment-format": {
+      "options": ["check-space"]
+    },
+    "curly": true,
+    "cyclomatic-complexity": false,
+    "eofline": true,
+    "forin": true,
+    "import-spacing": true,
+    "indent": {
+      "options": ["spaces"]
+    },
+    "interface-name": {
+      "options": ["always-prefix"]
+    },
+    "interface-over-type-literal": true,
+    "jsdoc-format": true,
+    "label-position": true,
+    "max-classes-per-file": { "options": 1 },
+    "max-line-length": { "options": 120 },
```

Comment:
If this is supposed to be lenient, should there be a max line length at all? Maybe a "ignore-pattern", maybe something like this for the main TS things
```
"ignore-pattern": "^import [^,]+ from |^export | implements"
```
This will ignore long imports & class definitions. Something to consider if it's useful for you

Reply:
Also, the base.json has 140 as max line length, so you should use 140 or more